packages/drivers/routerlicious-driver/src/summaryTreeUploadManager.ts

```diff
@@ -0,0 +1,160 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
```

Comment:
can you please give some info on why this is cloned? Is this to break any server / client dependency? Or something else?
local driver also uses it, can we put it into driver-utils such that all drivers can leverage same code?

Reply:
Because it uses gitmanager which we have changed, and it became incompatible. So, needed to be clone.

Reply:
I will do the deduping between local driver and r11s driver in separate PR.

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/4078